pepys_import/core/store/data_store.py

```diff
@@ -2066,3 +2070,23 @@ def edit_items(self, items, edit_dict):
                     new_value=str(getattr(item, col_name)),
                     change_id=change_id,
                 )
+
```

Comment:
@BarisSari - would you mind documenting the parameters for this method please? In particular, is `id_list` the ids of the items that are to be deleted?

I see it's not being called, but I guess that's because it's going to be called from Robin's UI code.  Aaah, since it's to be used elsewhere, it's probably worth documenting (commenting) the structure that gets returned.

I was going to suggest that you cover the new method with tests, but now realise this PR is still in Draft.

I probably shouldn't even be adding these comments/observations yet.

Oops 🤣

Reply:
Hello @IanMayo . Yes, that's the list of items that are to be deleted.

I'll document it and add tests. I just thought maybe I could wait until we get an answer from the client. That's why I only implemented a method that returns the dependent objects now. So, I'll add tests, don't worry. 😄